SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -286,13 +287,14 @@ void USpatialReceiver::CreateActor(Worker_EntityId EntityId)
 			}
 		}
 
-		// Update interest on the entity's components after receiving initial component data (so Role and RemoteRole are properly set).
-		//NetDriver->GetSpatialInterop()->SendComponentInterests(Channel, EntityId.ToSpatialEntityId());
-
-		// This is a bit of a hack unfortunately, among the core classes only PlayerController implements this function and it requires
-		// a player index. For now we don't support split screen, so the number is always 0.
+		// Only do the following on a client
```

Comment:
Maybe use `!NetDriver->IsServer()` to be explicit in the code and not have the comment.